src/pages/import/components/TimetableCourseCard.tsx

```diff
@@ -0,0 +1,182 @@
+import { useMemo, useState } from 'react';
+
+import { ChevronDownIcon, ChevronUpIcon } from '@chakra-ui/icons';
+import { Box, Flex, VStack, Text, Skeleton, IconButton, Collapse, HStack } from '@chakra-ui/react';
+import { Link as RouterLink } from 'react-router-dom';
+
+import { CourseDetails, Term, TimetableCourse, useGetCourse } from 'lib/fetchers';
+import { useDarkMode } from 'lib/hooks/useDarkMode';
+import { SavedCourse } from 'lib/hooks/useSavedCourses';
+
+import { useGetCourseSections } from 'pages/scheduler/hooks/useCalendarEvents';
+import { SavedCourseWithSections } from 'pages/scheduler/shared/types';
+
+function SectionInfoRow({ term, courses }: { term: Term; courses: SavedCourse[] }): JSX.Element {
+  const coursesResult = useGetCourseSections(term, courses);
+  const course: SavedCourseWithSections = useMemo(() => {
+    return coursesResult.status === 'loaded' && coursesResult.data
+      ? coursesResult.data[0]
+      : { ...courses[0], sections: [] };
+  }, [coursesResult, courses]);
+  console.log(coursesResult);
+
+  const sharedSections = useMemo(
+    () =>
+      course.sections.filter(
+        (c) => c.sectionCode === course.lecture || c.sectionCode === course.lab || c.sectionCode === course.tutorial
+      ),
+    [course]
+  );
+  // const sections = useMemo(() => {
+  //   coursesResult.status === 'loaded' && coursesResult.data ?
+  //   coursesResult.data[0].sections : []}, [coursesResult])
+
+  // const sectionTypes = useMemo(() =>
+  // Array.from(new Set(sections.map((s) => s.sectionType))), [sections]);
+
+  const mode = useDarkMode();
+  return (
+    <>
+      {coursesResult.status === 'loaded' && coursesResult.data ? (
+        <Box bgColor={mode('white', 'dark.main')}>
+          {sharedSections.map((s) => {
+            return (
+              <HStack
+                px="3"
+                my="0.5"
+                fontSize="12px"
+                borderTop={mode('light.background', 'dark.background')}
+                borderTopWidth="2"
+                borderTopStyle="solid"
+              >
+                <Text as="strong">{s.sectionCode}</Text>
+                <VStack flexGrow={1} py="1.5">
+                  {s.meetingTimes.map((m, key) => (
+                    <HStack key={key} w="100%" px="5">
+                      <Box w="33%" minW="27%">
+                        {m.time.split('-').map((time) => (
+                          <Text key={time}>{time}</Text>
+                        ))}
+                      </Box>
+                      <Box w="20%" minW="13%">
```

Comment:
Days probably doesn't need 20% of the space, I'd play around with all these a touch to find what makes best use of the space